lib/src/pbdl/pbdl_project.dart

```diff
@@ -34,7 +35,8 @@ class PBDLProject extends PBDLNode implements PBDLNodeFactory {
   List<SharedStyle> sharedStyles = [];
   PBDLGlobalStyles globalStyles = PBDLGlobalStyles();
 
-  int get materialDesignCount => MaterialDesign().totalMaterialCount;
+  int materialDesignCount(designSystem) =>
+      DesignSystemThemeData.getDesignSystem(designSystem).totalMaterialCount;
```

Comment:
Because this creates a whole new instance, is it possible to instead give PBDL the current `DesignSystemThemeData` so that we can just use its instance to get the count?

Reply:
you got it